article/app/controllers/ArticleController.scala

```diff
@@ -161,7 +161,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   def renderArticle(path: String) = {
     LongCacheAction { implicit request =>
-      mapModel(path) {
+      mapModel(path, request.isEmail) {
```

Comment:
Yep that makes sense. Just wondering what does Amp do as it has to come through `renderArticle` too? Does Amp use blocks @NataliaLKB @johnduffell? We should keep the methods for using `mapModel` with blocks consistent?